src/util/logger.js

```diff
@@ -0,0 +1,22 @@
+import pino from 'pino';
+
+
+const isDev = process.env.NODE_ENV !== 'production';
```

Comment:
This is a pretty common idiom in Node code, and I'd argue that everything outside of `'production'` is development of some description, but I take your point. I'll change this to `prettyPrint` so that it's more clear what it's being used for